client/app/api/course/Assessment/SubmissionQuestions.js

```diff
@@ -12,6 +12,17 @@ export default class SubmissionQuestionsAPI extends BaseAssessmentAPI {
     return this.getClient().post(`${this._getUrlPrefix()}/${submissionQuestionId}/comments`, params);
   }
 
+  /**
+  * Gets the past answers from a SubmissionQuestion
+  * Can include answers_to_load in params to indicate how many to pull (default 10)
+  *
+  * @param {number} submissionQuestionId
+  * @return {Promise}
+  */
+  getPastAnswers(submissionQuestionId) {
```

Comment:
Would suggest allowing an optional param to specify how many answers to load, and default it to 10.